apps/coscrad-frontend-e2e/src/e2e/resources/videos/video.index.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,204 @@
+import { AggregateType, LanguageCode } from '@coscrad/api-interfaces';
+
+const dummyVideoTitle = 'Dinosaur Video Title';
+
+const buildAggregateCompositeIdentifier = (id: string) => ({
+    type: AggregateType.video,
+    id: `9b1deb4d-3b7d-4bad-9bdd-2b0d7b100${id}`,
+});
+
+const aggregateCompositeIdentifier = buildAggregateCompositeIdentifier('001');
+
+const validUrl =
+    'https://coscrad.org/wp-content/uploads/2023/05/Rexy-and-The-Egg-_3D-Dinosaur-Animation_-_-3D-Animation-_-Maya-3D.mp4';
+
+// TODO[https://www.pivotaltracker.com/story/show/185717928] seed data with a title translation and unskip this test
```

Comment:
@bsellars Has completed the `TRANSLATE_VIDEO_NAME` command now. You should be able to do this. Just note that  you may need to add a test command fsa in `buildVideoTestCommandFsas`.